config/prod/bridgeworker.yaml

```diff
@@ -1,19 +1,19 @@
 template_path: templates/bridgeworker.yaml
-stack_name: bridgeworker-{{ environment_variable.TRAVIS_BRANCH }}
+stack_name: bridgeworker-prod
 parameters:
-  AwsDefaultVpcId: {{ environment_variable.AwsDefaultVpcId_prod }}
-  AwsSnsNotificationEndpoint: {{ environment_variable.AwsSnsNotificationEndpoint_prod }}
-  BridgeEnv: {{ environment_variable.BridgeEnv_prod }}
-  BridgeUser: {{ environment_variable.BridgeUser }}
-  BridgeWorkerEmail: {{ environment_variable.BridgeWorkerEmail_prod }}
-  BridgeWorkerStudy: {{ environment_variable.BridgeWorkerStudy_prod }}
-  BridgeWorkerPassword: {{ environment_variable.BridgeWorkerPassword_prod }}
+  AwsDefaultVpcId: vpc-9c70bbf9
+  AwsSnsNotificationEndpoint: bridgeworker-prod@sagebase.org
```

Comment:
Can we change the notification endpoint to bridgeops@sagebase.org? It's easier for me to monitor one mailing list instead of 3.

Reply:
Yes,  we can do that although I think having separate endpoints provides more flexibility for users to choose just the specific notifications they want.  Again i'm not opposed to it but I would prefer to do it in a separate PR.